nengo_viz/static/viz.js

```diff
@@ -127,6 +139,18 @@ VIZ.Component.prototype.on_resize = function(width, height) {};
  */
 VIZ.Component.prototype.on_message = function(event) {};
 
+
+VIZ.Component.prototype.generate_menu = function() {
+    var self = this;
+    var items = [];
+    if (this.label_visible) {
+        items.push(['hide label', function() {self.hide_label();}]);
+    } else {
+        items.push(['show label', function() {self.show_label();}]);    
+    }
+    return items;
```

Comment:
Yeah, I could see doing each item as a dictionary instead of as a list.  that sort of change will probably start to be necessary as we allow more complex menu stuff (like disabled items and checkboxes and whatever else we end up wanting).